webapp/commonComponents/validationFieldMessages.scss

```diff
@@ -0,0 +1,22 @@
+@import "~@webapp/style/vars";
+
+.validation_field_message {
```

Comment:
we use dash as separator for main css class ```validation-field-message```

Reply:
changed